src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration.mdx

```diff
@@ -1,21 +1,25 @@
 ---
-title: Amazon API Gateway monitoring integration
+title: Amazon API Gateway V1 monitoring integration
```

Comment:
@rahul188 the pattern I'm seeing across the docs is to us lower case like this: "v1"

Can we switch to that?

Reply:
@rhetoric101, I made the suggested change.